packages/next/src/server/request/cookies.ts

```diff
@@ -67,6 +67,11 @@ export function cookies(): Promise<ReadonlyRequestCookies> {
         throw new Error(
           `Route ${workStore.route} used "cookies" inside a function cached with "unstable_cache(...)". Accessing Dynamic data sources inside a cache scope is not supported. If you need this data inside a cached function use "cookies" outside of the cached function and pass the required dynamic data in as an argument. See more info here: https://nextjs.org/docs/app/api-reference/functions/unstable_cache`
         )
+      } else if (workUnitStore.phase === 'after') {
+        throw new Error(
+          // TODO(after): "close over it" is bad phrasing
+          `Route ${workStore.route} used "cookies" inside "unstable_after(...)". This is not supported. If you need this data inside an "unstable_after" callback, use "cookies" outside of the callback and close over it. See more info here: https://nextjs.org/docs/app/api-reference/functions/unstable_after`
```

Comment:
I think we can just drop the bad phrasing and still have it make sense.
```suggestion
          `Route ${workStore.route} used "cookies" inside "unstable_after(...)". This is not supported. If you need this data inside an "unstable_after" callback, use "cookies" outside of the callback. See more info here: https://nextjs.org/docs/app/api-reference/functions/unstable_after`
```

Reply:
yeah that sounds okay actually, i think i was trying to squeeze extra guidance into the error message but that should just go into docs instead